src/main/java/com.tourguide.tourguide2/TourGuideController

```diff
@@ -0,0 +1,19 @@
+package com.tourguide.tourguide2;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class TourGuideController {
+
+    /**
+     * Handle the root (/) endpoint and return a start page
+     * @return
+     */
+    @GetMapping ("/")
+    public String index() {
+        return "start";
+    }
+
+}
```

Comment:
Similarly to the TourGuideApplication, there was no TourGuideController. I added an example of what it could be. I was also unable to add in a proper java class, to which I will discuss in the larger code review comment.

Reply:
What problem did you have in adding a proper Java class?

Reply:
My issue was that I wasn't able to make a proper package stemming from the java directory that I had created. Looking into it more, it seems what I should have done but hadn't is mark the java directory as a sources root. Then it would have allowed me to add in the proper java class.